docs/source/i18n/ultimate.es.rst

```diff
@@ -2,7 +2,7 @@
 
 Spanish
 ~~~~~~~
-(translation courtesy of María de la Iglesia Vayá & Nicolas Pascual Leone Espinosa)
+(translation courtesy of María de la Iglesia Vayá, Nicolas Pascual Leone Espinosa & Guiomar Niso)
 
 
 Single access type version (all data shared publicly; recommended)
```

Comment:
I've seen versions in other languages don't translate this, but in case it's needed:


```suggestion
Single access type version (all data shared publicly; recommended)
Versión de acceso único (todos los datos son compartidos públicamente; recomendado)
```

Reply:
indeed would be nice to have it also in the native tongue.  Having with English probably also would not hurt. Thank you!